wasm/src/lib.rs

```diff
@@ -246,23 +360,7 @@ mod tests {
     use super::*;
 
```

Comment:
Hmm, could we use different allocator here, to emulate what happens between host and smartcontract? Just a thought.

Reply:
Hm... I don't know... Probably @mversic could help

Reply:
> Hmm, could we use different allocator here, to emulate what happens between host and smartcontract? Not a suggestion, just a thought.

can you paraphrase?

Reply:
@mversic, sure, @Arjentix, faced bug when he tried to deallocate inside iroha memory allocated by smartcontract. 
And our tests can't identify this because mocks are using the same global allocator as the tests themself i suppose.

Reply:
@Erigara , so you want to introduce a test that will check for allocation/deallocation problems with Iroha and WASM?

Reply:
No, i talking about using custom allocator inside mocks.